sidekiq-transaction-guard.gemspec

```diff
@@ -0,0 +1,36 @@
+# frozen_string_literal: true
+
+lib = File.expand_path("../lib", __FILE__)
+$LOAD_PATH.unshift(lib) unless $LOAD_PATH.include?(lib)
+require "sidekiq_transaction_guard/version"
+
+Gem::Specification.new do |spec|
+  spec.name          = "sidekiq-transaction-guard"
+  spec.version       = SidekiqTransactionGuard::VERSION
+  spec.authors       = ["Brian Durand", "Winston Durand"]
+  spec.email         = ["bbdurand@gmail.com", "me@winstondurand.com"]
+
+  spec.summary       = "Protect from accidentally invoking Sidekiq jobs when there are open database transactions"
+  spec.homepage      = "https://github.com/bdurand/sidekiq-transaction-guard"
+  spec.license       = "MIT"
+
+  # Specify which files should be added to the gem when it is released.
+  # The `git ls-files -z` loads the files in the RubyGem that have been added into git.
+  spec.files         = Dir.chdir(File.expand_path('..', __FILE__)) do
+    `git ls-files -z`.split("\x0").reject { |f| f.match(%r{^(test|spec|features)/}) }
+  end
+  spec.bindir        = "exe"
+  spec.executables   = spec.files.grep(%r{^exe/}) { |f| File.basename(f) }
+  spec.require_paths = ["lib"]
+
+  spec.required_ruby_version = '>= 2.0'
+
+  spec.add_dependency "activerecord", ">= 4.0", "< 6"
```

Comment:
We are not directly using `transaction_manager` in `ActiveRecord`. The methods we are using are delegates to `transaction_manager` but are not marked `#nodoc` so I'd consider them public. Thus, I think we can remove the upper bound on `activerecord`. There's always a chance any dependency can break on any release and it's better IMO to not lock out future versions for no reason.